index.bs

```diff
@@ -145,6 +145,8 @@ cache whenever the user clears their cookies or when session cookies expire.
 Note: A site can clear the browser's `Accept-CH` cache for its origin by sending an empty `Accept-CH`
 header in a response. This sets the origin's [=/client hints set=] to an empty set.
 
+Note: There MUST only be one `Accept-CH` header per-request and behavior is not defined if multiple are present.
```

Comment:
Why not define the behavior we expect servers to show if (when?) some clients would send such headers?